parachain/pallets/control/Cargo.toml

```diff
@@ -50,7 +47,11 @@ std = [
 	"sp-io/std",
 	"sp-runtime/std",
 	"xcm/std",
+	"xcm-builder/std",
```

Comment:
Seems `xcm-builder` is only used in test codes and not requied here.

Reply:
this failed when running `cargo test --features runtime-benchmarks` so I had to add this line.

Reply:
also require remove `default-features = false` in
`xcm-builder = { git = "https://github.com/paritytech/polkadot.git", branch = "master", default-features = false }` above

Already merged into `proxy-contracts` and seems CI tests for `runtime-benchmarks` just works fine.